tests/performance/load/groups/alerts.js

```diff
@@ -0,0 +1,40 @@
+import {Client, StatusOK} from 'k6/net/grpc';
+import {check, group} from 'k6';
+
+const GRPC_PORT = ":443";
+
+const client = new Client();
+
+client.load(['proto/stackrox', 'proto/googleapis'], 'api/v1/alert_service.proto');
+
+export function alertsGrpc(host, headers, tags) {
+
+    host = host.replace('https://', '')
+    host = host.includes(':') ? host : host + GRPC_PORT;
+    client.connect(host);
+
+    group('list alerts grpc', function () {
+    [0, 10, 100, 1000].forEach(limit => {
+        tags.limit = limit;
```

Comment:
I think this will change the original tags object. That means properties set in this function will be used by other functions executed after `alertsGrpc.`

i.e. (you can try in browser console)
```
a = {"a":"aaa"}
(function(tags){tags.limit=10; console.log(JSON.stringify({tags: tags}))})(a) // {"tags":{"a":"aaa","limit":10}}
console.log(JSON.stringify(a)) // {"a":"aaa","limit":10}
---
a = {a:"aaa"}
(function(tags){console.log(JSON.stringify({tags:{...tags, limit: 10}}))})(a) // {"tags":{"a":"aaa","limit":10}}
console.log(JSON.stringify(a)) // {"a":"aaa"}
```

The same applies to `tags.fetched`.